strawberryfields/backends/gaussianbackend/ops.py

```diff
@@ -17,105 +17,22 @@
 from itertools import tee
 
 import numpy as np
+
+from hafnian.quantum import density_matrix_element, is_pure_cov, pure_state_amplitude
+
 from scipy.linalg import sqrtm
 from scipy.special import binom, factorial
 
-
-def fock_amplitudes_one_mode(alpha, mat, cutoff, tol=1e-8):
+def fock_amplitudes_one_mode(alpha, cov, cutoff):
```

Comment:
@nquesada all 300 lines of removed code were to give the fock amplitudes? now handled by hafnian lib?

Reply:
yup. We were essentially implementing a really slow hafnian in ops. Now everything is done by single line calls to the hafnian library